packages/serverpod_shared/lib/src/util.dart

```diff
@@ -8,3 +8,14 @@ String generateRandomString([int length = 32]) {
   var values = List<int>.generate(length, (int i) => _random.nextInt(256));
   return base64Url.encode(values).substring(0, length);
 }
+
+/// Splits at spaces and joins to lowerCamelCase.
+String databaseTypeToLowerCamelCase(String databaseType) {
```

Comment:
I think there is a good dart library for doing these transformations. It may already be included in the CLI and/or elsewhere. Better to reuse than to rewrite, if possible.

Reply:
I've texted you about that. It's super_string. I've just not been sure if it's ok to add an extra dependency just for that.
I'll add it now.

Reply:
Better? 4b8a6171

Reply:
Yes, better keep things consistent across packages.